scripts/run-example.sh

```diff
@@ -55,6 +52,11 @@ function run_npm {
     cat package.json | jq '.devDependencies.turbo = "canary"' | sponge package.json
   fi
 
+  echo "======================================================="
+  echo "=> $folder: move lockfile"
+  echo "======================================================="
+  mv package-lock.json package-lock.json.bak
```

Comment:
I have a _mild_ preference for `rm` and then `git checkout` to restore it, but not a blocking level of preference. Your call.